local-tests/tests/wrapped-keys/util.ts

```diff
@@ -1,7 +1,50 @@
 import { LIT_NETWORKS_KEYS } from '@lit-protocol/types';
 import { LIT_CHAINS } from '@lit-protocol/constants';
 import { ethers } from 'ethers';
-import { EthereumLitTransaction } from '@lit-protocol/wrapped-keys';
+import { config } from '@lit-protocol/wrapped-keys';
+import {
+  litActionRepositoryCommon,
+  litActionRepository,
+} from '@lit-protocol/wrapped-keys-lit-actions';
+
+import type {
+  LitActionCodeRepository,
+  LitActionCodeRepositoryCommon,
+  EthereumLitTransaction,
+} from '@lit-protocol/wrapped-keys';
+
+const emptyLitActionRepositoryCommon: LitActionCodeRepositoryCommon = {
+  batchGenerateEncryptedKeys: '',
+};
+
+const emptyLitActionRepository: LitActionCodeRepository = {
```

Comment:
Sadly we can't run all of the tests twice (once w/ cids and once w/ actions) because this is a global mutation and it would require runInBand=true to avoid inconsistent behaviour.  When we migrate to Jest, I'll make that happen (since Jest sandboxes suites)

Reply:
Run them sequentially?

Reply:
We'd have to configure the entire suite to always run _all_ tests to `runInBand` the way `local-tests` is currently designed; since we're already in the midst of migrating to Jest and it will be able to run them in parallel it didn't seem like the effort was justified to enhance the existing local-tests code to allow subsets of tests to run in band